### PR TITLE
Delete stdout msgs

### LINKED ISSUE #1161: Delete messages printed to standard output when running simulations.
The messages printed in the terminal need to be improved in some ways and improved in others when running RuFaS simulations.

## Current behavior
When running RuFaS simulations, the current message(s) printed to standard out look like this:
```sh
$ python main.py -i -c
RuFaS: Ruminant Farm Systems Model 2023
Simulating...
/
Simulation Successful

/
Simulation Successful

/
Simulation Successful

$
```

## Proposed Improvements
1) It's 2024 now, not 2023.
2) After a simulation finishes successfully, it does not immediately go into the next simulation, it writes all the output files first. So repeating the "Simulating..." message when a new simulation actually starts would be helpful.
3) Either get the spinning progress wheel to work or get rid of it.
4) Add the simulation prefix from the `MetadataPaths` into the "Simulating..." message.

### PR DESCRIPTION
**Pull request by ew3361zh**
PR to remove the stdout write messages and add a log verbosity to print the credits.

### Context

Issue(s) closed by this pull request: closes #1161

- [x] The [changelog](https://docs.google.com/spreadsheets/d/1U-3igxdstHTFb6k5dVcE5-x9IL0r6q18eUiwwytKZyA/edit#gid=0) has been updated.

### What

1. Update LogVerbosity class to include a member for printing the credits (i.e. Ruminant Farm Systems...). Date has been removed from credit line.
2. Changes default LogVerbosity to credits.
3. Updates stdout messages to logs sent to OM (which will in turn link them with the metadata prefix used in the scenario in which they are logged).
4. Adds function in OM to print credits when LogVerbosity is any setting but `None`.
5. Removes progress spinning wheel from printing out to terminal. Not replaced with another progress bar - was a little complicated and because the Time class is being refactored soon, I think that will be better suited to be paired with that task #1099.
6. Updates unit testing.

### Why

These changes will reduce clutter printing out in the terminal during a default simulation. Logging the messages that used to print out will make it clearer as to which metadata scenario file they are being run from (which will make it easier to keep track of simulations of sequential scenarios).

### How

The LogVerbosity class was slightly refactored to accomodate the new member and is hopefully clearer as to what's happening with the different members including one another. The rest is fairly straightforward.

### Test plan

Testing was developed for the new OM function `print_credits()` and updated across simulation_engine, main, and output_manager.

### CODE REVIEW

File: RUFAS/output_manager.py [hunk 1]
```diff
@@ -1220,3 +1227,11 @@ def get_error_and_warning_counts(self) -> tuple[int, int]:
...
         warnings_count = sum([len(value_dict["values"]) for value_dict in self.warnings_pool.values()])
         return errors_count, warnings_count
+
+    def print_credits(self) -> None:
+        """
+        Prints out the RuFaS credits when LogVerbosity is set to any level except None.
+        """
+        if self.__log_verbose >= LogVerbosity.CREDITS:
+            current_year = date.today().year
+            sys.stdout.write(f"RuFaS: Ruminant Farm Systems Model {current_year}\n")
```
Comment (Isgotkowitz) [inline RUFAS/output_manager.py]: Would it maybe be better to not print the date at all? Right now there isn't anything 2024 specific to this "version" of RuFaS. But to me, it almost seems to be implied that this is the 2024 version of RuFaS.

File: main.py [hunk 2]
```diff
@@ -428,7 +422,9 @@ def execute_simulations(
         output_manager.set_metadata_prefix(metadata_file["prefix"])
         is_data_valid = input_manager.start_data_processing(str(metadata_file["path"]), True)
         if is_data_valid:
-            output_manager.add_log("Validation complete", "Data is valid. \nSimulating...\n", info_map)
+            output_manager.add_log(
+                "Validation complete", f"Data is valid. \nSimulating...{metadata_file['prefix']}" " scenario", info_map
```
Comment (Isgotkowitz) [inline main.py]:
```suggestion
                "Validation complete", f"Data is valid. Simulating {metadata_file['prefix']} scenario.", info_map
```
I think the log message looks a little cleaner like this.

**Review by Isgotkowitz — APPROVED** [review]
Looks great, nice job Niko.

File: RUFAS/output_manager.py (hunk 1 above)
Comment (ew3361zh) [reply RUFAS/output_manager.py]: True, I can remove that. Maybe we eventually put a version number if anything.

File: main.py (hunk 2 above)
Comment (ew3361zh) [reply main.py]: Updated!

**Review by ltp23 — APPROVED** [review]
It looks good to me, Niko!

I tested the different log verbosity levels and they worked as expected. I like that now we have more control over what is printed out to the console.

**Pull request merged**